k8s/overlays/azure/database.properties

```diff
@@ -0,0 +1,6 @@
+driver=postgresql
+host=<TO COMPLETE>
+database=directory
```

Comment:
same we shouldn't have the database set to directory here

Reply:
database property removed from file.